GitUI/OsShellUtil.cs

```diff
@@ -33,7 +48,7 @@ public static void OpenUrlInDefaultBrowser(string url)
         {
             if (!string.IsNullOrWhiteSpace(url))
             {
-                Process.Start(url);
+                new Executable(url).Start(useShellExecute: true);
```

Comment:
This may not work in .NET, despite [the docs](https://docs.microsoft.com/en-us/troubleshoot/dotnet/csharp/start-internet-browser) saying it should, and having it working in .NET Framework, I stumble across it yesterday in fact.

I had to add the `verb` option to get it working:

```diff
-            using var process = new Process
-            {
-                EnableRaisingEvents = false,
-                StartInfo = { FileName = uri.AbsoluteUri }
-            };
-            process.Start();
+            var ps = new ProcessStartInfo(uri.AbsoluteUri)
+            {
+                UseShellExecute = true,
+                Verb = "open"
+            };
+            Process.Start(ps);
         }
 }
```